spec/models/spree/gateway/amazon_spec.rb

```diff
@@ -74,6 +84,19 @@ def stub_auth_request(expected_body: nil, return_values: nil)
         expect(response).not_to be_success
         expect(response.message).to eq('Authorization failure: InvalidPaymentMethod')
       end
+      it 'updates last amazon transaction' do
```

Comment:
empty line missing - see to install [sublime linter](https://github.com/SublimeLinter/SublimeLinter-rubocop/issues/38) for those issues